models/sync-media/syntax.md

```diff
@@ -0,0 +1,67 @@
+# Synchronized Media Syntax
+
+This document introduces a new JSON based syntax for synchronizing media together.
+
+## Model
+
+The new syntax is based around a single JSON object called a Sync Media Object:
+
+| Name | Description | Format |
+| ---- | ----------- | ------ |
+| `audioref` | Points to a media fragment in an audio resource. | URI |
+| `children` | Array of media overlay nodes. | Sync Media Objects |
+| `imgref` | Points to a media fragment in an image resource. | URI |
+| `role`     | Array of roles relevant for the current node. | Array of roles |
+| `text`  | Text equivalent for the current Sync Media Object. | String |
+| `textref`  | Points to a fragment id in an HTML/XHTML resource. | URI |
+| `videoref`  | Points to a media fragment in an video resource. | URI |
+
+Each Sync Media Object MUST contain:
+
+- a `children` object containg at least one Sync Media Object
+- or at least two of the following elements: `audioref`, `imgref`, `text`, `textref` or `videoref`
+
+## Roles
+
+…
+
+## Examples
+
+*Example 1: Simple Sync Media Object with text and audio synchronized together*
+
+```json
+{
+  "textref": "chapter1.html#par1",
+  "audioref": "chapter1.mp3#t=0,20"
+}
+```
+
+*Example 2: Text equivalent of a speech bubble in an image*
+
+```json
+{
+  "role": ["speech-bubble"]
+  "imgref": "page10.jpg#xywh=percent:10,10,20,20",
+  "text": "This is a dialogue in a speech bubble."
+}
+```
+
+
+*Example 3: Chapter containing multiple Sync Media Objects*
+
+```json
+{
+  "textref": "chapter1.html",
```

Comment:
What's the purpose of the `xref`s in grouping elements? Does it limit the children to a given scope?